client/pool.c

```diff
@@ -1215,6 +1215,7 @@ static void print_connection(FILE *out, int index, struct connection_pool_data *
 		connection_state_to_string(conn_data->state), ip_port_str, in_out_str, connection_calculate_unpaid_shares(conn_data));
 }
 
+int print_connections(FILE *out);
```

Comment:
Do we need this declaration?

Reply:
Yes, please find the build error from AnotherYou. It's 
"
pool.c:1218:5: warning: no previous prototype for ‘print_connections’ [-Wmissing                                                                             -prototypes]
 int print_connections(FILE *out)
"